tests/Requester.py

```diff
@@ -263,6 +263,40 @@ def testBaseUrlPrefixRedirection(self):
             "Following Github server redirection from /api/v3/repos/PyGithub/PyGithub to /repos/PyGithub/PyGithub"
         )
 
+    def testMakeAbsoluteUrl(self):
+        # create a Requester with non-default arguments
+        requester = github.Requester.Requester(
+            auth=github.Auth.AppAuth(123, "key"),
+            base_url="https://base.url",
+            timeout=1,
+            user_agent="user agent",
+            per_page=123,
+            verify=False,
+            retry=3,
+            pool_size=5,
+            seconds_between_requests=1.2,
+            seconds_between_writes=3.4,
+            lazy=True,
+        )
```

Comment:
I presume we can use a default `Requester` here.
```suggestion
        requester = github.Requester.Requester()
```